sjb/config/test_cases/test_branch_origin_extended_conformance_install_containerized.yml

```diff
@@ -88,10 +93,11 @@ extensions:
     - type: "script"
       title: "ensure built version of origin is installed"
       timeout: 600
-      repository: "origin"
+      repository: "aos-cd-jobs"
       script: |-
-        origin_package="$( source hack/lib/init.sh; os::build::rpm::format_nvra )"
-        rpm -V "${origin_package}"
```

Comment:
In fact we are [building rpms](https://github.com/openshift/aos-cd-jobs/blob/master/sjb/config/test_cases/test_branch_origin_extended_conformance_install_containerized.yml#L13) but @ingvagabund is right that we are not using them since we are using built containers, that we build in "build an origin release" step